tendrl/commons/flows/import_cluster/__init__.py

```diff
@@ -13,6 +14,28 @@ def __init__(self, *args, **kwargs):
     def run(self):
         if "Node[]" not in self.parameters:
             integration_id = self.parameters['TendrlContext.integration_id']
+            short_name = self.parameters.get('Cluster.short_name', None)
+            if short_name:
+                if not re.match('^[a-zA-Z0-9][A-Za-z0-9_]*$', short_name):
+                    raise FlowExecutionFailedError(
+                        "Invalid cluster short_name: %s. "
+                        "Only alpha numeric and underscore "
+                        "allowed for short name" %
+                        short_name
+                    )
+            # Check for uniqueness of cluster short name
+            _clusters = NS._int.client.read(
+                '/clusters'
+            )
+            for entry in _clusters.leaves:
+                _cluster = NS.tendrl.objects.Cluster(
+                    integration_id=entry.key.split('/')[-1]
+                ).load()
+                if _cluster.short_name and short_name and \
```

Comment:
covert to lower case and check? remove the leading space if any as well

Reply:
done